advisories/github-reviewed/2024/12/GHSA-gmj6-6f8f-6699/GHSA-gmj6-6f8f-6699.json

```diff
@@ -1,18 +1,14 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-gmj6-6f8f-6699",
-  "modified": "2024-12-26T20:09:16Z",
+  "modified": "2024-12-26T20:09:18Z",
   "published": "2024-12-23T17:54:12Z",
   "aliases": [
     "CVE-2024-56201"
   ],
   "summary": "Jinja has a sandbox breakout through malicious filenames",
-  "details": "A bug in the Jinja compiler allows an attacker that controls both the content and filename of a template to execute arbitrary Python code, regardless of if Jinja's sandbox is used.\n\nTo exploit the vulnerability, an attacker needs to control both the filename and the contents of a template. Whether that is the case depends on the type of application using Jinja. This vulnerability impacts users of applications which execute untrusted templates where the template author can also choose the template filename.",
+  "details": "A bug in the Jinja compiler allows an attacker that controls both the content and filename of a template to execute arbitrary Python code, regardless of if Jinja's sandbox is used.\n\nTo exploit the vulnerability, an attacker needs to control both the filename and the contents of a template. Whether that is the case depends on the type of application using Jinja. This vulnerability impacts users of applications which execute untrusted templates where the template author can also choose the template filename.\n\nJinja before 3.0.0a1 is not vulnerable as it does not use f-strings for formatting error messages.",
```

Comment:
Don't include version information here, it's already explained by the version information fields.